resources/listen.rb

```diff
@@ -22,8 +22,19 @@
       variables['listen'][new_resource.name] ||= {}
       variables['listen'][new_resource.name]['mode'] ||= ''
       variables['listen'][new_resource.name]['mode'] << new_resource.mode
-      variables['listen'][new_resource.name]['bind'] ||= ''
-      variables['listen'][new_resource.name]['bind'] << new_resource.bind
+      if new_resource.bind.is_a? Hash
+        variables['listen'][new_resource.name]['bind'] = []
```

Comment:
The input data yes, should be `{}` but for correct render I'll prefer manage an ordered `[]`. Is it a problem?